pupil-spa/gen_config.sh

```diff
@@ -7,8 +7,7 @@ scriptDir="$( cd "$( dirname "$0" )" >/dev/null && pwd )"
 envFile=${scriptDir}/../.env
 [[ -f ${envFile} ]] && source ${envFile}
 applicationInsightsCode=${APPINSIGHTS_INSTRUMENTATIONKEY}
-authPingUrl=${AUTH_PING_URL:-"http://localhost:3003/ping"}
-authUrl=${AUTH_URL:-"http://localhost:3003/auth"}
+apiBaseUrl=${API_BASE_URL:-"http://localhost:3003"}
```

Comment:
This might be duplication of config values.  In admin app this env var is `PUPIL_API_BASE_URL`.

Reply:
modified